### PR TITLE
HFA-124

### PR DESCRIPTION
**Pull request by RandomUserIK**
- remove TODO on href elements, implement as router navigation
- add NOSONAR comments to ignore calls of bypassSecurityTrustUrl() method
- remove empty scss files
- display correct message when individual/interior projects are not found

### CODE REVIEW

File: src/app/public-view/components/footer/footer.component.ts [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
   selector: 'app-footer',
   templateUrl: './footer.component.html',
   styleUrls: ['./footer.component.scss']
 })
 export class FooterComponent {
+
+  constructor(private _router: Router) {
+  }
+
+  onLinkClick(): void {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/footer/footer.component.ts]: public by som ešte pridal

File: src/app/public-view/components/project/components/common-project/common-project-interior/common-project-interior.component.ts [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+  templateUrl: './common-project-interior.component.html'
 })
 export class CommonProjectInteriorComponent {
 
   @Input() projectImagePaths: Array<string>;
 
+  constructor(private _router: Router) {
+  }
+
+  onLinkClick(): void {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/project/components/common-project/common-project-interior/common-project-interior.component.ts]: Tu tiež public

File: src/app/public-view/components/project/components/common-project/common-project-price-comparison/common-project-price-comparison.component.ts [hunk 3]
```diff
@@ -10,4 +11,11 @@ export class CommonProjectPriceComparisonComponent {
 
   @Input() project: CommonProject;
 
+  constructor(private _router: Router) {
+  }
+
+  onLinkClick(): void {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/project/components/common-project/common-project-price-comparison/common-project-price-comparison.component.ts]: Tu tiež public

File: src/app/public-view/components/project/components/general/project-customer-request/project-customer-request.component.ts [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
   styleUrls: ['./project-customer-request.component.scss']
 })
 export class ProjectCustomerRequestComponent {
+
+  @Input() projectTitle: string;
+
+  constructor(private _router: Router) {
+  }
+
+  onButtonClick(): void {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/project/components/general/project-customer-request/project-customer-request.component.ts]: Tu tiež public

File: src/app/public-view/components/project/components/general/project-customized-proposal/project-customized-proposal.component.ts [hunk 5]
```diff
@@ -1,9 +1,18 @@
...
   selector: 'app-project-customized-proposal',
   templateUrl: './project-customized-proposal.component.html',
   styleUrls: ['./project-customized-proposal.component.scss']
 })
 export class ProjectCustomizedProposalComponent {
+
+  constructor(private _router: Router) {
+  }
+
+  onLinkClick(): void {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/project/components/general/project-customized-proposal/project-customized-proposal.component.ts]: Tu tiež public

**Review by MartinMiklovic — APPROVED** [review]

**Review by trofoSK — APPROVED** [review]

**Pull request merged**